### PR TITLE
chore(proto,types,node,grpc)!: Use `tendermint-rs` instead of `celestia-tendermint-rs` fork

### PR DESCRIPTION
**Pull request by oblique**

### CODE REVIEW

**Comment by oblique** [comment]
Stuff we need to check:

- [ ]  https://github.com/eigerco/celestia-tendermint-rs/issues/12
- [ ] All `tendermint::hash::Hash` deserialization in our crates. Because of [this](https://github.com/eigerco/celestia-tendermint-rs/pull/7).
- [ ] `Header::last_commit_hash`/`Header::last_results_hash` of block height 1. Because of this [this](https://github.com/eigerco/celestia-tendermint-rs/pull/4).
- [ ] Redefine `Block` in `celestia_proto::tendermint_celestia_mods` because of [this](https://github.com/eigerco/celestia-tendermint-rs/pull/1/files#diff-0e247661b40172c3f016f55a92ca66f24fda27f2dd07a7b29e663fbb2e3ec9dbL42).

File: proto/src/serializers/bytes.rs [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        T: AsRef<[u8]>,
+    {
+        let base64_bytes = base64::encode(value.as_ref());
+        let base64_string = String::from_utf8(base64_bytes).map_err(serde::ser::Error::custom)?;
+        serializer.serialize_some(&base64_string)
+    }
+}
+
+/// Serialize into string, deserialize from string
+pub mod string {
```
Comment (zvolin) [inline proto/src/serializers/bytes.rs]: please remove this one if unused, it sounds not very practical

Comment (oblique) [reply proto/src/serializers/bytes.rs]: done

File: proto/vendor/tendermint-celestia-mods/abci/types.proto [hunk 2]
```diff
@@ -0,0 +1,165 @@
+syntax = "proto3";
```
Comment (zvolin) [inline proto/vendor/tendermint-celestia-mods/abci/types.proto:1]: as I understand files in celestia-mods are modified by hand. Could you please write some note in readme / echo in update-proto-vendor what steps should be taken to maintain them?

**Review by zvolin — APPROVED** [review]
nice :bow:

**Review by fl0rek — APPROVED** [review]
nice work!

**Pull request merged**